package.json

```diff
@@ -84,6 +86,7 @@
     "build:watch": "webpack --watch",
     "test": "karma start --single-run",
     "test-debug": "cross-env NODE_ENV=debug karma start --no-single-run",
+    "test-coverage": "cross-env COVERAGE=true karma start --single-run",
```

Comment:
Let's simplify this and just always run coverage, whenever we run tests?

Reply:
I added this for couple of reasons,
1. circle-ci: but now we want it to fail, all good here (need to increase size config.yml `NODE_OPTIONS=--max_old_space_size=4096`)
1. `npm test` is lot quicker than `npm run test-coverage`, so its nice to have option for quick pass/fail check

Reply:
Or may be I can add another `npm run test-wo-coverage` and let `npm test` be with coverage?

Reply:
@nikhilmandlik It's fine to keep them how they are. Good point re: performance. Could we rename `test-coverage` to `test:coverage`? To keep it consistent with other build targets. Let's also renamed `test:debug` while we're at it.

Reply:
@akhenry Do we need, circle ci to use `npm run test:coverage` instead of `npm run test`?

Reply:
> Do we need, circle ci to use npm run test:coverage instead of npm run test?

Yes, please.